database/src/migrations/20221210155800_SIMS.1.5.0.ts

```diff
@@ -0,0 +1,326 @@
+import { Knex } from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+const DB_SCHEMA_DAPI_V1 = process.env.DB_SCHEMA_DAPI_V1;
+
+/**
+ *
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    set search_path=${DB_SCHEMA};
+
+    alter table project_attachment 
+      add column foi_reason_description varchar(3000),
+      add column government_interest_description varchar(3000);
+    comment on column project_attachment.foi_reason_description is 'The description of the Freedom of Information reason for securing of the artifact.';
+    comment on column project_attachment.foi_reason_description is 'The description of the government interest reason for securing of the artifact.';
```

Comment:
@cgarrettjones Looks like the comment here (and in the below alter statements) add a comment for the same column twice.

Reply:
adjusted.

Reply:
beauty!